srtcore/api.h

```diff
@@ -234,6 +234,9 @@ friend class CRendezvousQueue;
 
    CUDTException* getError();
 
+
+   CEPoll& epollmg() { return m_EPoll; }
```

Comment:
I would suggest to simply use `epoll()` function to return reference to CEPoll.

Reply:
I was thinking about it, but this name could be ambiguous. The intention is not to do any epoll on the object, but to return the reference to the internal epoll container. `epoll_ref()` maybe could be a better choice.

Reply:
Yes, `epoll_ref()` would also make sense.